targets/targets.json

```diff
@@ -9183,31 +9132,6 @@
         },
         "program_cycle_s": 10
     },
-    "CY8CKIT_062_WIFI_BT": {
```

Comment:
Why remove this target in commit "Refactor Mbed PSA service code base "?

Reply:
The target shouldn't have been removed. Fixed.